x-pack/legacy/plugins/siem/package.json

```diff
@@ -6,20 +6,37 @@
   "license": "Elastic-License",
   "scripts": {
     "build-graphql-types": "node scripts/generate_types_from_graphql.js",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run --reporter mocha-multi-reporters --reporter-options configFile=./reporter_config.json; mochawesome-merge --reportDir ../../../../target/kibana-siem/cypress/results > ../../../../target/kibana-siem/cypress/results/output.json; marge ../../../../target/kibana-siem/cypress/results/output.json --reportDir ../../../../target/kibana-siem/cypress/results; mkdir -p ../../../../target/junit && cp ../../../../target/kibana-siem/cypress/results/*.xml ../../../../target/junit/"
   },
   "devDependencies": {
     "@cypress/webpack-preprocessor": "^4.1.0",
     "@types/js-yaml": "^3.12.1",
     "@types/lodash": "^4.14.110",
     "@types/react-beautiful-dnd": "^10.0.1",
-    "cypress": "^3.3.1",
+    "cypress": "^3.4.1",
```

Comment:
Can you remind me the reason why we need to install this here instead of in `x-pack/package.json`? Is it just to make sure the `cypress` bin is available for the scripts?

Reply:
If we used `node scripts/cypress` to launch cypress we wouldn't need to worry about installing it here, could keep the `node_modules` shared amongst x-pack, and it would be more similar to how we do things elsewhere.

Reply:
We added `cypress` as a dev dependency of the `siem` app since we're the first project under `x-pack` to use it, but happy to move it up to `x-pack`. 😄

I'll also create a script for running it on the command line.